### ISSUE TITLE
Accessibility Enhancement: Allow Esc to close the filter dialog window

### ISSUE THREAD

**Issue by charmarkk** [sean-public/RecipeFilter#34]
I primarily use a keyboard to navigate the web, and it would be much simpler for me (particularly on longer recipes or recipe sites, like [this one from Just One Cookbook](https://www.justonecookbook.com/grilled-kurobuta-pork-chops-with-miso-sauce/)) to use the Esc key to *also* close the RF dialog window. Is this possible?

**Comment by sean-public** [comment]
That's a good idea! Should it also have `aria-live="assertive"` for screen readers as well?

**Comment by charmarkk** [comment]
I appreciate your enthusiasm. ☺️ I'd have to do some testing to see how it behaves as is. Very rarely do you want to use an assertive announcement (they're usually for urgent errors or alerts), as they interrupt the flow of the screen reader's default behavior. 

In most cases, when a dialog opens, keyboard focus should be set inside it and not leave/exit the dialog until the user closes the dialog. (Here's the [ARIA example pattern](https://w3c.github.io/aria-practices/examples/dialog-modal/dialog.html) for some nitty gritty. ☺️)

Very happy to see what I can find out to improve the accessibility of this extension and its forks. ☺️

**Comment by charmarkk** [comment]
Making the recipe dialog popup screen reader accessible will probably take some more thinking (it's not easily accessible as is). But adding the keyboard accessible to start with will be good!
